server/app/queries/populate-data.ts

```diff
@@ -1,12 +1,12 @@
 export const data =
 `SET search_path = QUIZ_DB;
 
-INSERT INTO Room (event_key, room_name, description, presenter_id) VALUES ('123456', 'Physics 101', 'Physics classroom', '21dn9ca82dniad3');
-INSERT INTO Room (event_key, room_name, description, presenter_id) VALUES ('abcdef', 'Maths 201', 'A place to have fun', '21dn9ca82dniad3');
+INSERT INTO Room (event_key, room_name, presenter_id) VALUES ('12345678', 'Physics 101', '21dn9ca82dniad3');
+INSERT INTO Room (event_key, room_name, presenter_id) VALUES ('abcdefgh', 'Maths 201', '21dn9ca82dniad3');
 
-INSERT INTO Quiz (max_duration, room_id) VALUES (300, 1);
-INSERT INTO Quiz (max_duration, room_id) VALUES (360, 2);
-INSERT INTO Quiz (max_duration, room_id) VALUES (300, 2);
+INSERT INTO Quiz (max_duration, title, room_id) VALUES (300, 'Last Week''s Lecture Recap', 1);
+INSERT INTO Quiz (max_duration, title, room_id) VALUES (360, 'Warm Up Quiz', 2);
+INSERT INTO Quiz (max_duration, title, room_id) VALUES (300, 'Class Recap', 2);
 
 INSERT INTO Question (question_label, correct_answer, quiz_id) VALUES ('Jane is using a 14 cm long screwdriver to assemble a bookcase. She applied a force of 45 N to rotate the screw 180 degrees. How much work did she do?', '19.79 J', 1);
```

Comment:
Perhaps into the populate data portion of the quiz, it would be helpful for you to also insert a few more types of choices for every question

Reply:
Will do